### DISCUSSION TITLE
Generated relation for tables to Files

### DISCUSSION THREAD

**Discussion by urskaufmann** [teksi/wastewater#34] [Q&A] [answered]
There are in lots of generated relations to "Files", everyone with a tab "Files" in drag and drop attribut form.
What are these relation for? Are they necessary? e.g. for class access_aid?

**Comment by sjib** [comment, marked as answer]
I think we should discuss whether the new Linking relation manager could be used instead
![linking_relation_editor](https://github.com/teksi/wastewater/assets/1265865/2cd6ad6b-f54d-4487-8733-ec58ebe575c5)

https://github.com/opengisch/qgis-linking-relation-editor

@3nids What do you think? Does this make things easier or more complicated?
